lib/CMakeLists.txt

```diff
@@ -102,36 +102,22 @@ add_compile_options($<$<CXX_COMPILER_ID:MSVC>:-MP>)
 include(GoogleTest)
 include(GenerateExportHeader)
 
-option(DISABLE_MULTI_THREADING "Disable multi threading" OFF)
-if(NOT DISABLE_MULTI_THREADING)
+option(THREADING "Build with multi-threading" ON)
+if(THREADING)
   if(SKBUILD)
     include(patch-pip-tbb-devel)
     find_package(TBB CONFIG REQUIRED)
     install(IMPORTED_RUNTIME_ARTIFACTS TBB::tbb RUNTIME DESTINATION
             ${PYTHONDIR}
     )
   else()
-    find_package(TBB CONFIG)
+    find_package(TBB CONFIG REQUIRED)
   endif()
 endif()
 if(TBB_FOUND)
```

Comment:
Does it still make sense to have a separate check for `TBB_FOUND`? Or can this `if-else` be merged with `if(THREADING)`?

Reply:
Done.